tests/core/pyspec/eth2spec/test/fulu/epoch_processing/test_process_proposer_lookahead.py

```diff
@@ -1,42 +1,31 @@
 from eth2spec.test.context import spec_state_test, with_fulu_and_later
+from eth2spec.test.helpers.epoch_processing import run_epoch_processing_with
 from eth2spec.test.helpers.state import next_epoch
 
 
 @with_fulu_and_later
 @spec_state_test
-def test_next_epoch_proposer_lookahead_shifted_to_front(spec, state):
-    """Test that the next epoch proposer lookahead is shifted to the front at epoch transition."""
-    # Transition few epochs to pass the MIN_SEED_LOOKAHEAD
-    next_epoch(spec, state)
-    next_epoch(spec, state)
+def test_proposer_lookahead_in_state_matches_computed_lookahead(spec, state):
+    """Test that the proposer lookahead in the state matches the computed lookahead."""
+    # Transition few epochs to past the MIN_SEED_LOOKAHEAD
+    for _ in range(spec.MIN_SEED_LOOKAHEAD + 1):
+        next_epoch(spec, state)
+
     # Get initial lookahead
     initial_lookahead = state.proposer_lookahead.copy()
 
     # Run epoch processing
-    yield "pre", state
-    next_epoch(spec, state)
-    yield "post", state
+    yield from run_epoch_processing_with(spec, state, "process_proposer_lookahead")
 
     # Verify lookahead was shifted correctly
     assert (
-        state.proposer_lookahead[: spec.SLOTS_PER_EPOCH]
+        state.proposer_lookahead[: spec.SLOTS_PER_EPOCH * spec.MIN_SEED_LOOKAHEAD]
```

Comment:
I think it's easier to read this
```suggestion
        state.proposer_lookahead[: -spec.SLOTS_PER_EPOCH]
```

Reply:
Yeah, I like this more :slightly_smiling_face: . Thanks!